x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plan/physical/EsSourceExec.java

```diff
@@ -62,7 +97,7 @@ protected NodeInfo<? extends PhysicalPlan> info() {
 
     @Override
     public int hashCode() {
-        return Objects.hash(index);
+        return Objects.hash(index, attributes, query, indexMode);
```

Comment:
Leaving them out was deliberate, see discussion in original PR. But I think this is now correct.

Reply:
Ah, cool.